sct.py

```diff
@@ -75,8 +75,8 @@
 from sdcm.utils.get_username import get_username
 from sdcm.send_email import get_running_instances_for_email_report, read_email_data_from_file, build_reporter, \
     send_perf_email
-from utils.build_system.create_test_release_jobs import JenkinsPipelines
-from utils.get_supported_scylla_base_versions import UpgradeBaseVersion
+from utils.build_system.create_test_release_jobs import JenkinsPipelines  # pylint: disable=no-name-in-module
+from utils.get_supported_scylla_base_versions import UpgradeBaseVersion  # pylint: disable=no-name-in-module
```

Comment:
You will be surprised, but there is utils package in the root of sct ))

Reply:
I am not surprised, I am saying that exactly it must be installed as package explicitly in such a case because it is out of the "sdcm" package. OR, place it under the "sdcm" dir. So, the concern is still valid.

Reply:
still not getting it, why other code does not need it to be installed, but this one needs it ?